doc/user_manual/code_interfaces/generic.tex

```diff
@@ -156,3 +156,90 @@ \subsection{Generic Interface}
   \item  \texttt{|}, the value that is going to be replaced by the Generic Interface, will be left- justified with a string length of ``10'';
 \end{itemize}
 
+The GenericCode interface also implements a mean for the user to stop a simulation while running if certain criteria are met, 
+monitoring the output file (CSV in this case). 
+Indeed, the user can provide a function to halt the simulation if a certain condition is met. The criterion or criteria is or are defined through
+an external RAVEN python function (e.g. ``criteria'' in the  following example). The time frequency of the check is defined
+by the node \xmlNode{onlineStopCriteriaTimeInterval} . By default, the output  is checked every 10 seconds.
+\\To activate such capability, the following addition XML nodes can be inputted:
+\begin{itemize}
+   \item \xmlNode{onlineStopCriteriaTimeInterval}, \xmlDesc{float, optional parameter}, time frequency (in seconds)) at which  the stopping function (if inputted)
+                                                            is inquired (stoppingCriteriaFunction). \default{10}.
+   \item \xmlNode{stoppingCriteriaFunction}, \xmlDesc{XML node, optional node}, the user can provide a
+                                                            function to halt the simulation if a certain condition is
+                                                            met. The criterion or criteria is or are defined through an external RAVEN python function. The function must return a ``bool'':
+                                                            \begin{itemize}
+                                                              \item True, if the simulation can continue (i.e. the criteria are not met)
+                                                              \item False, if the simulation must STOP (i.e. the criteria are met)
```

Comment:
I may prefer return "True" when the criteria are met here.

Reply:
perfect fixed.